src/test/java/com/g4s8/ghman/bot/TkCloseIssueTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.bot;
+
+import com.g4s8.ghman.user.Users;
+import com.jcabi.github.Repos;
+import com.jcabi.github.mock.MkGithub;
+import com.jcabi.matchers.XhtmlMatchers;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.Throws;
+import org.mockito.Mockito;
+import org.telegram.telegrambots.api.objects.CallbackQuery;
+import org.telegram.telegrambots.api.objects.Message;
+import org.telegram.telegrambots.api.objects.Update;
+
+/**
+ * Test for {@link TkCloseIssue}.
+ * @since 1.0
+ */
+final class TkCloseIssueTest {
+
+    @Test
+    void closesIssue() throws IOException {
+        final Update upd = Mockito.mock(Update.class);
+        final CallbackQuery clbq = Mockito.mock(CallbackQuery.class);
+        final String user = "world";
+        final String repo = "city";
+        final MkGithub github = new MkGithub(user);
+        github.repos().create(new Repos.RepoCreate(repo, false))
+            .issues().create("illuminate traffic", "");
+        Mockito.when(clbq.getMessage()).thenReturn(Mockito.mock(Message.class));
+        Mockito.when(clbq.getData()).thenReturn(
+            String.format(
+                "click:notification.close?repo=%s/%s&issue=%d", user, repo, 1
+            )
+        );
```

Comment:
@olenagerasimova isn't there any facility in teletake to test things without relying on mocking everything by hand? this test is very complex I find... maybe at least add a todo to improve this. See also #12.

Reply:
@victornoel unfortunately, for now there is no way for now to avoid mocking `Update`, teletakes has issue to create an abstraction https://github.com/g4s8/teletakes/issues/10. I do not want to add todo here, we can create separate issue when problem in teletakes will be resolved.